data/gfswind2png.py

```diff
@@ -103,13 +103,13 @@ def build_meta_json(data_dir, datetime, width, height, umin, umax, vmin, vmax):
         "vMax": round(vmax, 2),
         "minzoom": 0,
         "maxzoom": 2,
-        "tiles": [f"{data_dir}/{{z}}/{{x}}/{{y}}.png"],
+        "tiles": [f"{{z}}/{{x}}/{{y}}.png"],
     }
 
 
 def write_json(data_dir, json_output):
-    with open(f"{data_dir}tile.json", "w") as f:
-        f.write(json.dumps(json_output, indent=4, separators=(",", ": ")))
+    with open(os.path.join(data_dir, "tile.json"), "w") as f:
```

Comment:
Really should have done this tbh. Also, if you haven't already come across it, checkout `pathlib` for object based path manipulation, python built-in as of python 3.4.